.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,87 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ 'jetty-10.[1-9]?[0-9].x', 'jetty-11.[1-9]?[0-9].x', 'jetty-12.[1-9]?[0-9].x' ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ 'jetty-10.[1-9]?[0-9].x', 'jetty-11.[1-9]?[0-9].x', 'jetty-12.[1-9]?[0-9].x' ]
+  schedule:
+    - cron: '22 1 * * 2'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'java', 'javascript' ]
```

Comment:
we do not have any javascript here. So there is no real need of a matrix

Reply:
We do have javascript, in the demo apps. Check the bottom of the "files changed" tab in this PR to see where it's finding bugs.

Reply:
ah yup demos 🤦